lib/Rx/SchedulerInterface.php

```diff
@@ -17,4 +17,11 @@ public function schedule($action);
      * @return DisposableInterface
      */
     public function scheduleRecursive($action);
+
+    /**
+     * Gets the current representation of now for the scheduler
+     *
+     * @return mixed
```

Comment:
@asm89 - The only use of the now() function right now is to allow a window inside the ReplaySubject. Returning seconds does make more sense. I have committed this. See https://github.com/davidwdan/Rx.PHP/commit/d742828b0b1c1665051363d8707fa2bac54f732e . I return microtime where when I can because it is apparently it is not available on every platform.